core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1289,6 +1289,13 @@ public String toString() {
                   + "(alluxio.master.embedded.journal.election.timeout).")
           .setDefaultValue("1s")
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE)
+          .setDescription(" Amount of data that is appended from leader to followers "
```

Comment:
NIT: empty space prefix

also, what is the unit?

Reply:
unit is BYTES. Updated the name to reflect that.